### ISSUE TITLE
Fix contrast and list elements on index.html

### ISSUE THREAD

**Issue by Spongebosch** [diamondmyracle/HW1#86]
Right now, the contrast on the tierlist is too low to pass some of the accessibility tests. Additionally, the <br> tags outside of the <li> tags fail some tests too. I may also try to see about fixing the headers that aren't ordered properly.

**Comment by Spongebosch** [comment]
did it
